public_html/wp-content/plugins/camptix-attendance/addons/assets/attendance-ui.css

```diff
@@ -274,6 +274,32 @@ li.item .spinner:after {
     font-size: 16px;
     padding-left: 2px;
     padding-right: 16px; }
+#attendees-list-wrapper .qr-scanner {
+	display: none;
+	z-index: 400;
+	position: absolute;
+	bottom: 1em;
+	right: 1em;
+	height: auto;
+	width: 90vw;
+	width: calc(100% - 2em);
+	max-width: 480px;
+}
+#qr-reader {
+	min-height: 200px;
+}
+#qr-reader__dashboard_section {
+	_display: none;
+	position: absolute;
+	background: #fff;
+	bottom: 0;
+}
+.qr-scanner:hover #qr-reader__dashboard_section {
+	display: block;
+}
+.qr-scanner-active #attendees-list-wrapper .qr-scanner {
+	display: block;
+}
```

Comment:
You've only added the new styles to the compiled `.css` file, but you should have added them to the `.scss` file instead and compile the `.css` file again.